vermouth/tests/test_edge_tuning.py

```diff
@@ -0,0 +1,553 @@
+        [-1.7400000095367432, -0.5400000214576721, 0.8500000238418579],
+        [0.4300000071525574, -1.309999942779541, -1.7899999618530273],
+        [-1.3300000429153442, 1.5399999618530273, -0.25],
+        [0.8799999952316284, -1.9299999475479126, 1.2899999618530273],
+        [1.7799999713897705, 1.5299999713897705, 1.690000057220459],
+        [-1.7100000381469727, 0.20999999344348907, 0.8199999928474426],
+        [0.07999999821186066, -1.1200000047683716, 1.399999976158142],
+        [-0.3199999928474426, -0.49000000953674316, 0.9599999785423279],
+        [0.7400000095367432, -1.8200000524520874, 1.1799999475479126],
+        [1.0399999618530273, -1.7000000476837158, -1.6399999856948853],
+        [0.4000000059604645, -1.0700000524520874, 1.25],
+        [1.2200000286102295, 1.440000057220459, -0.6399999856948853],
+        [-0.47999998927116394, -1.3799999952316284, 1.3200000524520874],
+        [-0.46000000834465027, -0.07000000029802322, -0.5199999809265137],
+        [0.05000000074505806, 0.5799999833106995, 1.659999966621399],
+        [1.090000033378601, -0.17000000178813934, -0.8600000143051147],
+        [1.159999966621399, 0.6499999761581421, 0.15000000596046448],
+        [-0.3400000035762787, 0.9200000166893005, -0.07000000029802322],
+        [0.17000000178813934, 1.2300000190734863, 0.23999999463558197],
+        [1.559999942779541, 1.5800000429153442, -0.3100000023841858],
+        [1.059999942779541, -0.6700000166893005, -0.4300000071525574],
+        [-1.059999942779541, -0.3499999940395355, 1.9299999475479126],
+        [-1.6299999952316284, 0.5299999713897705, -0.5299999713897705],
+        [1.3799999952316284, -1.2699999809265137, 0.7400000095367432],
+        [-1.4900000095367432, 1.6799999475479126, 0.41999998688697815],
+        [1.159999966621399, -1.809999942779541, 1.2000000476837158],
+        [-0.17000000178813934, -0.28999999165534973, 1.7899999618530273],
+        [-1.9500000476837158, -1.25, 1.2200000286102295],
+        [-0.05000000074505806, 0.8899999856948853, -1.350000023841858],
+        [1.5099999904632568, 1.3300000429153442, 0.11999999731779099],
+    ])
+
+
+@pytest.fixture
+def multi_molecules(coordinate_array):
+    """
+    Build 6 molecules with 6 atoms each. Coordinates are populated unter the
+    `coords` attribute key with the coordinates produced by
+    :func:`coordinate_array`.
+    """
+    molecules = []
+    for _ in range(6):
+        molecule = vermouth.molecule.Molecule()
+        molecule.add_nodes_from([(idx, {'atomid': idx, 'resid': 1})
+                                 for idx in range(6)])
+        molecules.append(molecule)
+    iter_nodes = (
+        node
+        for molecule in molecules
+        for node in molecule.nodes.values()
+    )
+    for idx, (node, coords) in enumerate(zip(iter_nodes, coordinate_array)):
+        node['coords'] = coords
+        node['serial'] = idx
+    return molecules
+
+
+class TestPruneEdgesBetweenSelection:
```

Comment:
The classes are there to group tests together and avoid having lost fixtures everywhere. Without the classes you get a litany of seemingly unrelated test functions. It could also latter on allow for simpler function names.